astropy/io/fits/tests/test_core.py

```diff
@@ -1184,7 +1185,7 @@ def test_write_bytesio(self):
 
         self._test_write_string_bytes_io(io.BytesIO())
 
-    @pytest.mark.skipif('sys.platform.startswith("win32")')
+    @pytest.mark.skipif(sys.platform.startswith("win32"), reason="Requires Unix")
```

Comment:
Slightly more accurate like this?
```suggestion
    @pytest.mark.skipif(sys.platform.startswith("win32"), reason="Cannot test on Windows")
```